### PR TITLE
Bean card

### PR DESCRIPTION
**Pull request by jakelong95**
Handles #66 and #81

### CODE REVIEW

File: src/components/dashboard/browse/BeanItemDetails.js [hunk 1]
```diff
@@ -46,6 +46,13 @@ class BeanItemDetails extends Component {
                     <div>Item ID: {this.props.bean.id}</div>
```
Comment (ghmeier) [inline src/components/dashboard/browse/BeanItemDetails.js]: users dont need to see an id

File: src/components/dashboard/browse/BeanItemDetails.js [hunk 2]
```diff
@@ -46,6 +46,13 @@ class BeanItemDetails extends Component {
...
+                    <div>{this.props.bean.ozInBag} oz in each bag</div>
+                    {this.props.bean.isDecaf && 
+                        <div>Decaf</div>
+                    }
+                    {this.props.bean.description != '' &&
+                        <div>{this.props.bean.description}</div>
+                    }                    
                     <h1> Roaster Details </h1>
                     <div>Name: {this.props.roaster.name}</div>
                     <div>Location: {this.props.roaster.addressLine1} - {(this.props.roaster.addressLine2 + ' ' || '') + this.props.roaster.addressState}
```
Comment (ghmeier) [inline src/components/dashboard/browse/BeanItemDetails.js]: this is not relevant to bean details

File: src/components/dashboard/browse/BeanItemDetails.js [hunk 3]
```diff
@@ -46,6 +46,13 @@ class BeanItemDetails extends Component {
                     <div>Item ID: {this.props.bean.id}</div>
                     <div>Price: {this.props.bean.consumerPrice}</div>
                     <div>In Stock: {this.props.bean.inStockBags}</div>
+                    <div>{this.props.bean.ozInBag} oz in each bag</div>
+                    {this.props.bean.isDecaf && 
+                        <div>Decaf</div>
+                    }
+                    {this.props.bean.description != '' &&
+                        <div>{this.props.bean.description}</div>
+                    }
```
Comment (ghmeier) [inline src/components/dashboard/browse/BeanItemDetails.js]: we can also add bean.tags which is an array of strings representing tags.

**Review by ghmeier — CHANGES_REQUESTED** [review]
I think there could be some more effort put into considering what users need to see rather than just listing the raw text from the user.

**Comment by jonnykry** [comment]
Should display the quantity (e.g., "1 lb"), too

**Review by ghmeier — APPROVED** [review]
Looks good. Nice to have the input validation.

Might want to look at jazzing up the styling a bit.

**Pull request merged**